WeatherApp/ViewControllers/WeatherListViewController.swift

```diff
@@ -7,14 +7,19 @@
 //
 
 import UIKit
+import RxSwift
+import RxCocoa
 import PureLayout
 
 class WeatherListViewController: UIViewController {
     
     private let weatherListView = WeatherListView()
     private let errorView = ErrorView()
     private var weatherViewModel: WeatherListViewModel!
-    
+    private let disposeBag = DisposeBag()
+    private let refreshControl = UIRefreshControl()
+    private var spinner = SpinnerViewController()
```

Comment:
same thing as with UITableView.  You can just place a UIActivityIndicatorView directly in this view controller. Wrapping it inside a ViewController doesn't add any value in this case.